app/components/therapist/TherapistAppointmentDeclineModal.js

```diff
@@ -0,0 +1,232 @@
+import React, { useState } from "react";
+import {
+  Modal,
+  StyleSheet,
+  Text,
+  View,
+  TouchableOpacity,
+  TextInput,
+  Button,
+  Alert,
+} from "react-native";
+import { BlurView } from "expo-blur";
+import Constants from "expo-constants";
+import { Formik } from "formik";
+import RNPickerSelect from "react-native-picker-select";
+import DateTimePicker from "@react-native-community/datetimepicker";
+import notificationsApi from "../../api/notifications";
+import bookingsApi from "../../api/bookings";
+
+const TherapistAppointmentDeclineModal = ({
+  item,
+  visible,
+  handleDeclineModal,
+}) => {
+  const reasonOptions = [
+    "Location out of range",
+    "Not available at that time",
+    "Other",
+  ];
+
+  const handleSubmit = async (values) => {
+    let convertDate = values.date.toLocaleDateString();
+    let convertTime = convertDateTimeToLocalTimeStr(values.time);
+    console.warn("convertTime", convertTime);
+    console.warn("convertDate", convertDate);
+
+    if (!values.reason) {
+      Alert.alert("Please select a reason"),
+        [
+          {
+            text: "Close",
+            onPress: () => {},
+          },
+        ];
+    } else {
+      // add some logic for sending athletes new appointment
+      let booking_status = await bookingsApi.declineBooking(
+        item.therapistData.bookings_id
```

Comment:
could you make it so the declineBooking service accepts a suggestedBookingDateTime parameter in the request body like this: 
![image](https://github.com/kevins-hub/sportstretch-ui-master/assets/25918539/47ad77cf-a606-441a-9c16-ed3454694e0b)

my bad I forgot to put this SS in the google doc smh. Everything looks good tho great work.